cmd/di_desktop.go

```diff
@@ -62,6 +61,10 @@ import (
 
 // bootstrapServices loads all the components required for running services
 func (di *Dependencies) bootstrapServices(nodeOptions node.Options, servicesOptions config.ServicesOptions) error {
+	if nodeOptions.MobileConsumer {
```

Comment:
What was wrong with build flags approach?
Because now file is di_desktop.go but full of conditional logic, which would be nice to avoid

Reply:
1. I can't build mobile entry point using `go build ./mobile/mysterium` since di_mobile will not be included but di_mobile doesn't contain any OS specific logic.
2. There are multiple checks already with feature flags, example https://github.com/mysteriumnetwork/node/blob/master/cmd/di_desktop.go#L342. 
3. I can't write e2e test properly. If I add GOOS=android it will require specific C++, but for now I just want to write e2e test which tests that mobile entry point doesn't crash on startup, so if connect is not working at least use can sent feedback.

Reply:
I too am for conditional compiling, if possible.

Reply:
Agree with @Waldz here. We should not increase our codebase with platform checks inside if we can build accordingly.

Reply:
80% of current mobile entrypoint API can be tested without any android specific build tag and most importantly it tests for possible crashes when some dependencies are not added to mobile . If you still want conditional compiling I can close this PR and add a separate task since it will require 3-4 days at least to make it work.

Reply:
So it's just for the test, but not released artefact, ASFAIK.
Is there a way to force a android tag in this test particularly. Maybe adding custom flag like "e2e" is what you need?